### PR TITLE
Add timeout for launch_cvd process

### PR DESCRIPTION
**Pull request by MHA3**
This process can sometimes hang, thereby preventing Clusterfuzz from proceeding to fuzz. We would like to kill the process so that Clusterfuzz can have another go at it.

### CODE REVIEW

File: src/clusterfuzz/_internal/platforms/android/adb.py [hunk 1]
```diff
@@ -378,7 +379,10 @@ def start_cuttlefish_device(use_kernel=False):
     launch_cvd_command_line += (
         f' -kernel_path={kernel_path} -initramfs_path={initramfs_path}')
 
-  execute_command(launch_cvd_command_line, on_cuttlefish_host=True)
+  execute_command(
+      launch_cvd_command_line,
+      timeout=LAUNCH_CVD_TIMEOUT,
```
Comment (oliverchang) [inline src/clusterfuzz/_internal/platforms/android/adb.py:384]: This just causes the `execute_command` to return None (https://github.com/google/clusterfuzz/blob/148f3e3a17e7f5223c5957137c1f48e557bbaedc/src/clusterfuzz/_internal/platforms/android/adb.py#L115), without adding any explicit retry. 

Should we be explicitly retrying instead here?

Comment (MHA3) [reply src/clusterfuzz/_internal/platforms/android/adb.py:384]: I discussed it with Cory. We have manually killed this process in the past which let Clusterfuzz proceed normally. So, just plain killing this process should be okay for us.

Comment (MHA3) [reply src/clusterfuzz/_internal/platforms/android/adb.py:384]: Clusterfuzz tries to reboot the device once the process is manually killed, and the second time it works.

**Review by oliverchang — APPROVED** [review]

**Pull request merged**